modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java

```diff
@@ -562,6 +571,16 @@ else if (Objects.equals(
 		}
 	}
 
+	private void _validateInvokerBundle(String messageType, boolean system)
```

Comment:
@pedro-oliveira446 Can you put it inside the ObjectDefinitionUtil and consume from there?

Reply:
Remember to adapt the code according to Brian suggestions